src/HuC1.md

```diff
@@ -1,50 +1,51 @@
 # HuC1
 
-HuC1 is an MBC used by some Game Boy games which besides doing the usual
-MBC stuff, also provides IR comms. A lot of sources on the internet said
-that HuC1 was \"similar to MBC1\", but they didn\'t provide any detail.
-I took a look, and it turns out that HuC1 differs from MBC1 quite a lot.
+HuC1 is an MBC used by some Game Boy games which, besides doing the
+usual MBC stuff, also provides infrared communication. A lot of sources
+on the internet said that HuC1 was \"similar to MBC1\", but they didn\'t
+provide any detail. I took a look, and it turns out that HuC1 differs
+from MBC1 quite a lot.
 
 ## Memory Map
 
 Address range | Feature
 --------------|------------------------------------
-  $0000-1FFF  | RAM/IR select (when writing only)
-  $2000-3FFF  | ROM bank select (when writing only)
-  $4000-5FFF  | RAM bank select (when writing only)
-  $6000-7FFF  | Nothing?
-  $A000-BFFF  | Cart RAM or IR register
+  $0000–1FFF  | RAM/IR select (when writing only)
+  $2000–3FFF  | ROM bank select (when writing only)
+  $4000–5FFF  | RAM bank select (when writing only)
+  $6000–7FFF  | Nothing?
+  $A000–BFFF  | Cart RAM or IR register
 
-### 0000-1FFF IR Select (Write Only)
+### 0000–1FFF IR Select (Write-Only)
```

Comment:
I appreciate the effort, but the section title does not follow [the style guide](https://github.com/gbdev/pandocs/blob/master/CONTRIBUTING.md#8-section-titles) anyway, so I am currently preparing a PR to fix that. (If you can follow it in the new docs, that would be very appreciated!)